### ISSUE TITLE
WIN32OLERuntimeError - failed to create WIN32OLE object from 'iTunes.Application'

### ISSUE THREAD

**Issue by awongCM** [sunny/so-nice#18]
Hi guys,

Now that we resolved the anyplayer issue with Itunes Windows not connected with anyplayer, I started to resume trying out sonice.

When I ran sonice at the command prompt (under Windows obviously :) ).  I encountered the following error logs written on screen.  (see attached)
![sonice-errors](https://cloud.githubusercontent.com/assets/1639795/3495184/00c7aa92-05d3-11e4-8407-fe3cf50a1d2d.png)

Hope you can see the error message..

Basically it says it failed to instantiate WIN32OLE object from my Itunes.  

But the odd thing about it is that I can still somewhat able to interact with my running Windows itunes using Sonice web interface ie can play/pause, move next/prev track.etc.  Only the response is very slow for itunes to accept the web interface controls commands.

Hope this makes sense.

Let me know if you need further info from me.
